packages/teleterm/src/mainProcess/clusterContextMenu.ts

```diff
@@ -0,0 +1,107 @@
+import { ipcMain, ipcRenderer, Menu } from 'electron';
+
+enum ClusterContextMenuEventChannel {
```

Comment:
Are we ok with enums?

Reply:
I would put all constants used for channel messages into the `/src/mainProcess/types.ts`. Using enums are OK.

Reply:
Done